pygfx/helpers/_axes.py

```diff
@@ -74,15 +74,12 @@ def __init__(self, size=1.0, thickness=2):
         for pos, color in zip(line_positions[1::2], colors[1::2]):
             material = MeshBasicMaterial(color=color)
             arrow_head = Mesh(cone, material)
-            arrow_head.position = Vector3(*pos)
+            arrow_head.local.position = pos
             # offset by half of height since the cones
             # are centered around the origin
-            arrow_head.position.add_scaled_vector(
-                Vector3(*pos).normalize(), arrow_size / 2
-            )
-            arrow_head.rotation.set_from_unit_vectors(
-                Vector3(0, 0, 1),
-                Vector3(*pos).normalize(),
+            arrow_head.local.position += arrow_size / 2 * la.vector_normalize(pos)
```

Comment:
`+=`? is that supported?

Reply:
Actually, it does. It ultimately translated to:

```python
arrow_head.local.position.__set__(arrow_head.local.position.__get__().__iadd__(arrow_size / 2 * la.vector_normalize(pos))
```

Which is a mouth full, but eventually calls `__set__` on the local position which updates the value as desired. It also modifies the cached position in-place, which you might think could influence future calls, but `__set__` invalidates the cache causing the value to be re-generated. 

Are you by chance thinking of slicing? This is unsupported, so `arrow_head.local.position[0] = 42` would not have any effect.

Reply:
Yeah, I think it's important to write these "reactivity gotcha's" down in the docs or guide.

Reply:
Sure. Where would users search for this information? Then I'll place it there to increase the chance of it being found.

Reply:
I'd say a subsection at the end of the Guide called "Updating transforms" would be useful. If people get stuck on transforms not updating because they're assigning to slices, that's probably going to be their search keyword.